File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        mapN={}
        lenN=len(nums)

        for i in range(lenN):
            mapN[nums[i]]=i

        for i in range(lenN):
            j=target - nums[i]
            if j in mapN and mapN[j] != i:
                return [i, mapN[j]]
        
        return []
        
        

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] res=new int[2];
        for(int i=0;i<nums.length;i++){
            for(int j=i+1;j<nums.length;j++){
                if(nums[i]+nums[j]==target) {
                    res=new int[]{i,j};
                    break;
                }
            }
        }
        return res;
    }
}